_posts/blog/2016-10-28-durability-performance.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Durability Performance Implications"
+date: 2016-10-28 17:00:00 +0000
+author: Keith Turner
```

Comment:
I was thinking it would be nice to add another optional field: `reviewers`.   For Fluo we decided not have an author field because the post were a community effort. As is the case here, review always made the post so much better.  I am thinking a different way to solve this than Fluo did is to add a reviewer field.  Unless someone objects, I can add handling for that in the template in this PR.